.github/workflows/tests.yml

```diff
@@ -21,6 +21,14 @@ jobs:
     - name: Test all
       run: ./script/util/make.sh test-all -j2
 
+  test-coverage:
+    runs-on: ubuntu-20.04
+    name: Coveralls
+    steps:
+    - uses: coverallsapp/github-action@master
+      with:
+        github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I don't think we need this in our CI. Users can check the test coverage using `go test -cover`.

cc @AkihiroSuda

Reply:
This thread might be helpful:  https://stackoverflow.com/questions/10516662/how-to-measure-test-coverage-in-go

Reply:
But thank you very much for your suggestion! :smile:

Reply:
@ktock Thank you for your feedback!